htdocs/app/Migrations/Version20170830005500.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
```

Comment:
lease remove this auto generated comment

Reply:
changed this to a meaningful summary